Sources/SwiftDocC/Semantics/MarkupReferenceResolver.swift

```diff
@@ -56,9 +61,12 @@ struct MarkupReferenceResolver: MarkupRewriter {
             // If the linked node is part of the topic graph,
             // verify that linking to it is enabled, else return `nil`.
             if let node = context.topicGraph.nodeWithReference(resolved) {
-                guard context.topicGraph.isLinkable(node.reference) else {
+                if !context.topicGraph.isLinkable(node.reference) {
                     problems.append(disabledLinkDestinationProblem(reference: resolved, source: source, range: range, severity: severity))
                     return nil
+                } else if node.isEmptyExtension {
```

Comment:
I guess not; writing a link like ``` ``Swift/Array#Topics`` ``` emitted a "'Topics' doesn't exist at 'Swift/Array'" warning instead of the "empty extension" one.

Reply:
`## Topics` and other topic sections are somewhat special headings that we don't support linking to. A custom heading should work.

Reply:
I think it would be hard to test something like this because without an extension file or any child symbols, the page is completely empty. Should i test with an arbitrary heading? I did try with ``` ``Swift/Array#Array`` ``` and got the "empty extension page" warning instead.

Reply:
I added a test that linked to an arbitrary heading to ensure that it still gets the "empty extension page" warning, but i'm not sure it's what you had in mind. Can you double-check for me?